shared/js/background/components/resource-loader.js

```diff
@@ -108,7 +108,7 @@ export default class ResourceLoader extends EventTarget {
                 await this._updateData(result);
                 break;
             } catch (e) {
-                console.warn('Load failed...', e);
+                // console.warn('Load failed...', e);
```

Comment:
Did you mean to leave this change in? Silently ignoring the failure seems like it would make tracking down issues harder?

Reply:
Yes - this path is commonly hit when we fall back to another mechanism to get the config (e.g. fetch fails, fallback to DB), so logging here is a bit noisy (particularly in tests).